pages/dkp/kommander/2.2/licensing/enterprise/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: layout.pug
+navigationTitle: DKP Enterprise
+title: DKP Enterprise
+excerpt: A section containing all of the features available to users with a DKP Enterprise license.
+menuWeight: 5
+beta: false
+---
+
+## DKP Enterprise overview
+
+![DKP Enterprise Diagram](/dkp/kommander/2.2/img/dkpenterprisediagramv2.png)
+
+DKP Enterprise is a multi-cluster lifecycle management Kubernetes solution centered around a management cluster that manages multiple attached or managed Kubernetes clusters via a centralized management dashboard. The management dashboard gives you a single point of observability and control throughout all of your attached or managed clusters. The DKP Enterprise license gives the user access to the entire Konvoy cluster environment, the Kommander UI dashboard that deploys platform and catalog applications, and multi-cluster management, and comprehensive compatibility with our full range of infrastructure deployment options.
+
+## Compatible infrastructure
+
+DKP Enterprise operates across D2iQ's entire range of cloud, on-premise, edge, and air-gapped infrastructures and has support for various OSs, including immutable OSs. See [Supported Operating Systems][supported] for a full list of compatible infrastructure.
+
+For the basics on standing up a DKP Enterprise cluster in one of the listed environments of your choice, see [Choose Infrastructure][choose-infrastructure].  
+
+## Platform applications
+
+Applications can be deployed in any Kubernetes-managed or self-attached clusters, giving you complete flexibility to operate across cloud, on-premise, edge, and air-gapped scenarios. DKP Enterprise users can use the Kommander UI to customize which platform applications to deploy to the cluster in a given workspace.
```

Comment:
`DKP Enterprise users can use the Kommander UI to customize which platform applications to deploy to the cluster in a given workspace.
`
Let's use DKP UI, as we are using Kommander UI here and DKP UI later in the docs, this leads to inconsistency.